packages/v4/src/content/design-guidelines/components/tree-view/tree-view.md

```diff
@@ -3,5 +3,92 @@ id: Tree view
 section: components
 ---
 
-We’re working on some guidelines for this component.
-If you have any questions in the meantime, feel free to [get in touch with us.](/get-in-touch)
+A **tree view** is a structure that displays data in a hierarchical view. It can be used in a [dropdown](/components/dropdown), [drawer](/components/drawer), [primary-detail](/demos/primary-detail), [modal](/components/modal), or [wizard](/components/wizard).
+
+A default tree view provides a tree structure with items and an arrow to expand or collapse child items. Every tree view can be extended with these [optional elements](#tree-view-capabilities).
```

Comment:
I would change this a little to say "with items and arrows to expand..."
Basically make arrow plural instead of singular, since there will be more than one arrow in a tree view structure!

Reply:
Also i would say "optional capabilities" instead of "elements" to match the header for that section